_includes/pbc/all/install-features/202212.0/install-the-product-configuration-feature.md

```diff
@@ -80,6 +80,36 @@ To make sure that the changes have been applied, check that the exemplary produc
 
 {% endinfo_block %}
 
+{% info_block infoBox %}
+
+You can control whether particular fields should be filtered out and not used for Product Configuration instance hash generation. You can do it via the `ProductConfigurationConfig::getConfigurationFieldsNotAllowedForEncoding()` config setting.
+
+{% endinfo_block %}
+
+**src/Pyz/Service/ProductConfiguration/ProductConfigurationConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Service\ProductConfiguration;
+
+use Generated\Shared\Transfer\ProductConfigurationInstanceTransfer;
+use Spryker\Service\ProductConfiguration\ProductConfigurationConfig as SprykerProductConfigurationConfig;
+
+class ProductConfigurationConfig extends SprykerProductConfigurationConfig
+{
+    /**
+     * @return list<string>
+     */
+    public function getConfigurationFieldsNotAllowedForEncoding(): array
+    {
+        return [
+            ProductConfigurationInstanceTransfer::QUANTITY,
```

Comment:
we need to at least mention that this transfer must be defined on a project level